.github/pull.yml

```diff
@@ -0,0 +1,14 @@
+version: "1"
+rules:
+  - base: master
+    upstream: UncleGoogle:master
+    mergeMethod: none  # Optional, one of [none, merge, squash, rebase, hardreset], Default: none.
+    mergeUnstable: false    # Optional, merge pull request even when the mergeable_state is not clean. Default: false
+    # assignees:              # Optional
```

Comment:
I'm not sure about practical difference reviewer vs assignees. 

Probably assignee should be a person responsible for the synchronizing between reviewers/ take final decisions and merge?

Reply:
That seems like a reasonable way to break apart those two roles, the assignee owns the work of making it across the finish line.

Reply:
> Probably assignee should be a person responsible for the synchronizing between reviewers/ take final decisions and merge?

Yeah and should probably be you as you know your way around the repo better than anyone else. 
```suggestion
    assignees:              # Optional
      - UncleGoogle
```

Reply:
Oh actually that makes a lot more sense than what I thought. I think leaving assignees blank is probably the best for now.